eq_cir_management_ui/__init__.py

```diff
@@ -0,0 +1,116 @@
+"""Flask application factory for the EQ CIR Management UI."""
+
+import json
+import logging
+import os
+from pathlib import Path
+
+from flask import Flask
+from flask_talisman import Talisman
+from jinja2 import ChainableUndefined, FileSystemLoader
+
+from eq_cir_management_ui.config.config import DefaultConfig
+from eq_cir_management_ui.errors.routes import errors_blueprint
+from eq_cir_management_ui.main.routes import main_blueprint
+from eq_cir_management_ui.utils.routes import utils_blueprint
+
+logger = logging.getLogger()
+
+talisman = Talisman()
+
+
+def create_app(app_config: type[DefaultConfig]) -> Flask:
+    """Flask application factory, used to isolate the instance of the Flask application.
+    See https://flask.palletsprojects.com/en/2.2.x/patterns/appfactories/ .
+    """
+    app = Flask(__name__)
+
+    app.config.from_object(app_config)
+    app.static_folder = str(Path("../static"))
```

Comment:
This dir doesn't exist for me but I'm not sure if I've configured it correctly?

Also, it might be more convenient to use an absolute path here instead of relative paths. So something similar like:
`app.static_folder = Path(__file__).parent / "static"`

Reply:
Static folder was configured but not added, as it is currently empty.

Have removed setting until static folder is needed